### PR TITLE
Feature lc 237 lc 291 display mulitple dates and times for events

### PR DESCRIPTION
**Pull request by KLV96**
**Jira Ticket:**
https://cshrdigitalandanalysis.atlassian.net/browse/LC-237

This branch has the implementations for displaying multiple dates on the following pages:

- Booking a course
- Booking a course
- Summary page

This branch also displays the start time and end time (part **a** and **b** of LC-291)

### CODE REVIEW

File: src/lib/datetime.ts [hunk 1]
```diff
@@ -151,6 +151,10 @@ export function formatTime(d: number | Date, timeOnly?: boolean) {
 	return timeFormat.format(d)
 }
 
+export function removeSeconds(t: any) {
+	return t.substring(0, t.length - 3);
```
Comment (kameshganesan) [inline src/lib/datetime.ts]: semicolon?

File: src/ui/page/booking/summary.html [hunk 2]
```diff
@@ -27,12 +27,24 @@ <h3 class="heading-small">
...
+                                {:else}
+                                <span class="vertical-center booking__key">Date and Time:</span>
+                                {/if}
+                            </div>
+
+                            <div class="date-and-time-value-container">
+                                <span class="booking__value">
+                                    {#each event.dateRanges as dateRange, index}
+                                        {$datetime.formatDate(dateRange.date)}  at {$datetime.removeSeconds(dateRange.startTime)} to {$datetime.removeSeconds(dateRange.endTime)}
+                                        <br>
```
Comment (kameshganesan) [inline src/ui/page/booking/summary.html]: <br/>

File: src/locale/en.json [hunk 3]
```diff
@@ -380,5 +380,34 @@
 	"line-manager": "line-manager",
 	"other-areas-of-work": "other-areas-of-work",
 	"Commercial": "Commercial",
-	"profile_primary-area-of-work": "profile_primary-area-of-work"
+	"profile_primary-area-of-work": "profile_primary-area-of-work",
+	"Associate Commercial Specialist": "Associate Commercial Specialist",
+	"Policy": "Policy",
+	"Commercial Support": "Commercial Support",
```
Comment (m24murray) [inline src/locale/en.json:386]: @KLV96 did you add these intentionally?

Comment (KLV96) [reply src/locale/en.json]: got removed. Thanks

**Pull request merged**